Cloud changelog - Dec 8 (#5290)
mattermost/docs: source/install/cloud-changelog.md

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

Latest Mattermost Cloud releases:
- [Release 2021-12-08](#release-2021-12-08)
- [Release 2021-12-01](#release-2021-12-01)
- [Release 2021-11-23](#release-2021-11-23)
- [Release 2021-11-11](#release-2021-11-11)
- [Release 2021-11-10](#release-2021-11-10)
- [Release 2021-10-27](#release-2021-10-27)

## Release 2021-12-08

### Improvements

#### User Interface (UI)
 - Added thread replies to search results when collapsed reply threads feature is enabled.
 - Updated "Terms of Service" terminology to "Terms of Use" product-wide.

>  - Government related references are now shown for Government type licenses.

Reviewer: ```suggestion
 - Government-related references are now shown for Government type licenses.
```

Author: Removed it as this is more relevant to on-prem.

#### Performance
 - Added a general performance fix for loading the web application and typing.

>  - Improved performance while typing by moving some autocomplete layout calculation.

Reviewer: ```suggestion
 - Improved performance while typing by moving some autocomplete layout calculations.
```

Reviewer: I'm not 100% sure that pluralizing the word "calculations" is technical accurate (where multiple calculations were impacted by a change); I pluralized it to make the sentence flow a bit better.

### Bug Fixes
 - Fixed slow channel loading for instances with website link previews enabled.

>  - Fixed an issue where the webapp crashed sometimes when clicking on an image file from "Recent files".

Reviewer: ```suggestion
 - Fixed an issue where the webapp crashed sometimes when selecting an image file from "Recent files".
```

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

Latest Mattermost Cloud releases:
- [Release 2021-12-08](#release-2021-12-08)
- [Release 2021-12-01](#release-2021-12-01)
- [Release 2021-11-23](#release-2021-11-23)
- [Release 2021-11-11](#release-2021-11-11)
- [Release 2021-11-10](#release-2021-11-10)
- [Release 2021-10-27](#release-2021-10-27)

## Release 2021-12-08

### Improvements

#### User Interface (UI)

>  - Added thread replies to search results when collapsed reply threads feature is enabled.

Reviewer: ```suggestion
 - Added thread replies to search results when Collapsed Reply Threads is enabled.
```

This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.

Latest Mattermost Cloud releases:
- [Release 2021-12-08](#release-2021-12-08)
- [Release 2021-12-01](#release-2021-12-01)
- [Release 2021-11-23](#release-2021-11-23)
- [Release 2021-11-11](#release-2021-11-11)
- [Release 2021-11-10](#release-2021-11-10)
- [Release 2021-10-27](#release-2021-10-27)

## Release 2021-12-08

### Improvements

#### User Interface (UI)
 - Added thread replies to search results when Collapsed Reply Threads is enabled.
 - Updated "Terms of Service" terminology to "Terms of Use" product-wide.

#### Performance
 - Added a general performance fix for loading the web application and typing.
 - Improved performance while typing by moving some autocomplete layout calculations.
 - Improved performance by reducing DOM usage during render.

>  - Removed real-time updates of a couple features to prevent overloading servers on user updates.

Reviewer: We may want to consider specifying which features no longer have real-time updates for clarity and transparency.

>  - Removed real-time updates of a couple features to prevent overloading servers on user updates.

Author: cc @hmhealey if you have more details on this.
